calculator/src/components/Inputs.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react'
+
+function Inputs({ calculateTip }) {
+
+  const [total, setTotal] = useState(0);
+  const [tip, setTip] = useState(0);
+
+  const handleChangeTotal = (e) => {
+    setTotal(Number(e.target.value));
+  }
+
+  const handleChangeTip = (e) => {
+    setTip(Number(e.target.value));
+  }
+
+  const handleClick = () => {
+    if (total < 0 || tip < 0) {
+      alert("Input has to be a number and it can't be negative!")
+    } else {
+      calculateTip(total, tip);
+    }
+  }
+
+  return (
+    <div>
+      <table>
```

Comment:
as this are inputs with a click button, maybe a Form would have more sense than a table.